### PR TITLE
Update 'Parallel SR-IOV configuration' sections

### PR DESCRIPTION
**Pull request by e0ne**

### CODE REVIEW

File: docs/getting-started-kubernetes.rst [hunk 1]
```diff
@@ -857,19 +857,87 @@ The ``pod.yaml`` configuration file for such a deployment:
 SR-IOV Network Operator Deployment – Parallel Node Configuration for SR-IOV
 ---------------------------------------------------------------------------
 
-.. warning:: This is a Tech Preview feature, which is supported only for Vanilla Kubernetes deployments with SR-IOV Network Operator.
+.. warning:: This is a feature is supported only for Vanilla Kubernetes deployments with SR-IOV Network Operator.
```
Comment (rollandf) [inline docs/getting-started-kubernetes.rst:860]: Suggestion:
This feature is supported only for....

File: docs/getting-started-kubernetes.rst [hunk 2]
```diff
@@ -857,19 +857,87 @@ The ``pod.yaml`` configuration file for such a deployment:
...
 
-To apply SriovNetworkNodePolicy on several nodes in parallel, specify the ``maxParallelConfiguration`` option in the SriovOperatorConfig CRD:
+To apply configure SR-IOV on several nodes in parallel, the ``SriovNetworkPoolConfig`` CRD and specify number or percent of unavailable nodes at the moment:
```
Comment (rollandf) [inline docs/getting-started-kubernetes.rst:862]: suggestion:
To apply SR-IOV configuration on several nodes in parallel, create a ``SriovNetworkPoolConfig`` CR and specify the maximum number or percentage of nodes that can be unavailable at the same time:

File: docs/getting-started-kubernetes.rst [hunk 3]
```diff
@@ -857,19 +857,87 @@ The ``pod.yaml`` configuration file for such a deployment:
...
+              - val-2
+        - matchExpressions:
+          - key: other-label
+            operator: "Exists"
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Upgrade from NVIDIA Network Operator v24.1.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To upgrade SR-IOV Network operator you need to create ``SriovNetworkPoolConfig`` CRD with the number of nodes to be configured in a parallel as we did in `SriovOperatorConfig`` in previous releases.
```
Comment (rollandf) [inline docs/getting-started-kubernetes.rst:910]: Suggestion:
``SriovNetworkPoolConfig`` CR

File: docs/getting-started-kubernetes.rst [hunk 4]
```diff
@@ -857,19 +857,87 @@ The ``pod.yaml`` configuration file for such a deployment:
...
+
+E.g.: old method to congigure nodes in a prarallel:
```
Comment (rollandf) [inline docs/getting-started-kubernetes.rst:912]: congigure -> configure
prarallel => parallel

File: docs/getting-started-kubernetes.rst [hunk 5]
```diff
@@ -857,19 +857,87 @@ The ``pod.yaml`` configuration file for such a deployment:
...
 .. code-block:: bash
 
-    kubectl patch sriovoperatorconfigs.sriovnetwork.openshift.io -n network-operator default --patch '{ "spec": { "maxParallelNodeConfiguration": 0 } }' --type='merge'
+    kubectl patch sriovoperatorconfigs.sriovnetwork.openshift.io -n network-operator default --patch '{ "spec": { "maxParallelNodeConfiguration": 5 } }' --type='merge'
+
+
+New  method to congigure nodes in a prarallel:
```
Comment (rollandf) [inline docs/getting-started-kubernetes.rst:918]: congigure -> configure
prarallel => parallel

File: docs/getting-started-kubernetes.rst [hunk 6]
```diff
@@ -857,19 +857,87 @@ The ``pod.yaml`` configuration file for such a deployment:
...
+        - matchExpressions:
+          - key: node-role.kubernetes.io/master
+                operator: Exists
 
 --------------------------------------------------------------------------
 SR-IOV Network Operator Deployment – Parallel NIC Configuration for SR-IOV
 --------------------------------------------------------------------------
 
-.. warning:: This is a Tech Preview feature, which is supported only for Vanilla Kubernetes deployments with SR-IOV Network Operator.
+.. warning:: This feature, which is supported only for Vanilla Kubernetes deployments with SR-IOV Network Operator.
```
Comment (rollandf) [inline docs/getting-started-kubernetes.rst:940]: suggestion:
This feature is supported only for Vanilla Kubernetes

File: docs/getting-started-kubernetes.rst (hunk 1 above)
Comment (e0ne) [reply docs/getting-started-kubernetes.rst:860]: done

File: docs/getting-started-kubernetes.rst (hunk 2 above)
Comment (e0ne) [reply docs/getting-started-kubernetes.rst:862]: done

File: docs/getting-started-kubernetes.rst (hunk 3 above)
Comment (e0ne) [reply docs/getting-started-kubernetes.rst:910]: done

File: docs/getting-started-kubernetes.rst (hunk 4 above)
Comment (e0ne) [reply docs/getting-started-kubernetes.rst:912]: done

File: docs/getting-started-kubernetes.rst (hunk 5 above)
Comment (e0ne) [reply docs/getting-started-kubernetes.rst:918]: done

File: docs/getting-started-kubernetes.rst (hunk 6 above)
Comment (e0ne) [reply docs/getting-started-kubernetes.rst:940]: done

**Review by rollandf — APPROVED** [review]

**Pull request merged**